modules/sql/src/main/scala/Slit.scala

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem.sql
+
+import java.math.RoundingMode.HALF_UP
+
+/** Minimal Slit classes to support reading from the enum tables.
+  *
+  * NOTE: For now this is a placeholder for slit dimensions replicating
+  * Wavelength. Scientifically it seems the fundamental unit should be
+  * arcseconds from which other representations could be derived. How to
+  * implement this properly should be clearer after we identify the commonality
+  * with other instruments or we come with a more general solution about
+  * handling units in general in gem.
+  */
```

Comment:
Perhaps it's not worth creating this placeholder now. Maybe it's better to put GNIRS implementation on hold and try to come up with a more permanent solution for the units problem now.